### DISCUSSION TITLE
Forcing operation's material

### DISCUSSION THREAD

**Discussion by matteocarcano** [frePPLe/frepple#602] [Q&A] [answered]
Hi, 
is it possible to forcing operation's material assignment?
For example, Manufacturing Order 235 of item A has item B in his Bill of material; after plan execution, MO 235 consumes 10 pieces of item B supllied by MO 999; can i manually change it with another MO of item B or force to use the stock?

Thank you for reply.

Bye

**Comment by jdetaeye** [comment, marked as answer]
Yes, you can. 
Typical use cases:
- An operation consumes items X and Y in its bill of material, but for whatever reason the manufacturing order is manually adjusted to consume X and Z instead.
- A manufacturing order needs to consume items X and Y.  The material X has already been issued to the manufacturing order, so we want the plan to consume only Y.    

The manufacturing needs to be in the status approved or confirmed, and you need to provide the "inventory detail" records as a separate data feed.
When processing such a manufacturing order, we'll just work with the inventory detail records you provided.
If no inventory detail records are found, frepple will automatically generate them itself based on the bill of material.

**Reply by matteocarcano** [reply]
Thank you for reply.

It's very clear but i don't undestand how can i manually change it. I see the inventory detail but i don't find the way to edit supply path.

**Comment by jdetaeye** [comment]
We don't have user-friendly screens to maintain that information in frepple.   

Frepple is a planning tool.
All inventory and productions operations are tracked in an ERP or MES system.
Interfaces bring the detailed progress information from ERP or MES into frepple for planning.

**Reply by matteocarcano** [reply]
Ok thank you, now It's clear.
